core/pfe-core/controllers/roving-tabindex-controller.ts

```diff
@@ -80,41 +112,64 @@ export class RovingTabindexController<
    */
   get prevItem(): ItemType | undefined {
     return (
-        this.#activeIndex > 0 ? this.#focusableItems[this.#activeIndex - 1]
-      : this.lastItem
+      this.#activeIndex > 0 ? this.#focusableItems[this.#activeIndex - 1]
+        : this.lastItem
     );
   }
 
   constructor(public host: ReactiveControllerHost & HTMLElement) {
```

Comment:
@bennypowers curious on what you mean here, could you elaborate?  Are you suggesting you could pass options that would allow you to skip the init such as a set of items and their container?

Reply:
correct. see the options signatures, you could make them lazy  (se floating dom controller) if they aren't immediately available

Reply:
Moving this one to https://github.com/patternfly/patternfly-elements/issues/2644 so we don't hold up listbox